content/akeneo-event-platform/getting-started.md

```diff
@@ -0,0 +1,159 @@
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "name": "example subscriber name",
+    "contact": {
+        "technical_email": "subscriber.test.email@example.com"
+    }
+}'
+```
+
+Response Example :
+```json
+{
+    "id": "01905a84-a3b7-766e-a49f-5519c35fa7a0",
+    "tenant_id": "pim_tenant",
+    "client_id": "3444ec1b-058e-4208-9b6c-284f47a7aa17",
+    "name": "subscriber name",
+    "subject": "https://pim.cloud.akeneo.com",
+    "status": "active",
+    "contact": {
+        "technical_email": "subscriber.email@example.com"
+    },
+    "created_at": "2024-06-27T16:26:00.503422Z",
+    "updated_at": "2024-06-27T16:26:00.503422Z"
+}
+```
+After creating the subscriber, store the id from the response in an environment variable:
+```bash
+export SUBSCRIBER_ID="01905a84-a3b7-766e-a49f-5519c35fa7a0"  # Replace with the actual ID from the response
+```
+
+### 3. Create a Subscription
+
+With a subscriber in place, the next step is to create a subscription to specify which events you want to receive.
+In this example, we will use an HTTPS destination (HTTPS is mandatory for secure communication).
+To create a subscription, you will need a destination URL.
+
+```bash
+export DESTINATION_URL="https://my-destination-url.com"  # Replace with your destination URL
+```
+
+::: info
+💡 You don’t have to worry about the secret part of the configuration for now, they are used to sign the payload (more information below in the API presentation section)
+:::
+
+```bash
+curl --request POST "https://event.prd.sdk.akeneo.cloud/api/v1/subscriber/$SUBSCRIBER_ID/subscription" \
+--header "X-PIM-URL: $TARGET_PIM_URL" \
+--header "X-PIM-TOKEN: $PIM_API_TOKEN" \
+--header "X-PIM-CLIENT-ID: $CLIENT_ID" \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "source": "pim",
+    "subject": "'"$TARGET_PIM_URL"'",
+    "events": [
+        "com.akeneo.pim.v1.product.updated"
+    ],
+    "type": "https",
+    "config": {
+        "url": "'"$DESTINATION_URL"'",
+        "secret": {
+            "primary": "averysecretprimarysecret",
+            "secondary": "you can leave empty here, only use to ease your secret rotation use cases"
+        }
+    }
+}'
+```
+
+### 4. Trigger an Event from the PIM
+
+With your subscription in place, you're ready to trigger the event you subscribed to (in this example, `com.akeneo.pim.v1.product.updated`). The simplest way to do this is by updating a product directly within the Akeneo PIM UI. This approach allows you to manually trigger the event and observe the changes in real-time.
+
+Alternatively, you can also update a product using the [Akeneo PIM REST API](/api-reference.html#post_products_uuid) if you prefer an automated approach.
+
+After the product is updated, verify that the event was successfully triggered by checking the payload received at your destination URL.
+
+::: panel-link Let's see the API reference! [Next](/akeneo-event-platform/api-reference.html) 
```

Comment:
Shouldn't we have a dedicated page with the direct link to the API Ref so we avoid sending the reader to a whole new section without the Summary as Arianne's thread?